packages/server/src/api/controllers/workspace.ts

```diff
@@ -175,6 +179,53 @@ async function createInstance(appId: string, template: AppTemplate) {
   return { _id: appId }
 }
 
+function getCreatorMetadataId(ctx: UserCtx) {
```

Comment:
I know there are other utility functions in here, so it's not a big deal. But I wonder if the controller is the right place for them. `packages/server/src/sdk/users/utils.ts` is probably the place.